app/views/ScreenLockConfigView.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Switch } from 'react-native';
+import Switch from '../containers/Switch';
 import { connect } from 'react-redux';
```

Comment:
path imports should be in the bottom and in sequential order so move redux import to 2nd line